bokeh/models/tools.py

```diff
@@ -52,10 +52,6 @@ class Tool(Model):
 
     '''
 
-    plot = Instance(".models.plots.Plot", help="""
-    The Plot that this tool will act on.
-    """)
-
```

Comment:
To be completely safe we should add a no-op python property that warns, at least for a little while, in case anyone is setting this by hand themselves.